### PR TITLE
Added scroll to account list

### PR DESCRIPTION
**Pull request by micaelgfilipe**
I have a lot of accounts, the addition of a scroll would be nice.

### CODE REVIEW

File: src/lib/content.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ function loadProfiles(profileSet, list, csrf, hidesHistory, hidesAccountId) {
   var AWSR_firstForm = null;
 
   document.getElementById('awsc-recent-roles-label').insertAdjacentHTML('beforeend', '<input id="AESR_RoleFilter" type="text" placeholder="Filter by profile name" style="border:1px solid #ccc;border-radius:3px;font-size:13px;margin-left:0.25em;max-width:20ex;padding:0.4ex">');
-
+  document.getElementById('awsc-username-menu-recent-roles').setAttribute("style", "height:200px;overflow:auto");
```
Comment (tilfin) [inline src/lib/content.js]: @micaelgfilipe We need examine whether the 200px is appropriate and whether the scroll-bar width causes the style broken in Windows browsers. The height should be ideally changed according to the browser screen height by `calc(100vh - <absolute Y of account list>)`.

Comment (micaelgfilipe) [reply src/lib/content.js]: I agree, and have improved. Regarding the possible Windows issue, I am not able to test it atm.

**Comment by tilfin** [comment]
Filter processing can break the bound of account item in Windows, it must be fixed.
An account item is tricky element because it is form tag not plain block tag.

image
Fi

**Comment by micaelgfilipe** [comment]
> Filter processing can break the bound of account item in Windows, it must be fixed.
> An account item is tricky element because it is form tag not plain block tag.
> 
> image

Which browser are you using?

**Comment by tilfin** [comment]
It's Chrome in Windows

**Comment by jimbocoder** [comment]
Can the container not simply be made a bit wider to accomodate this on Windows?

Unfortunately I can't help test this on Windows, but I want to add my support for this PR after a fix is found. Scrolling will be really helpful so I don't have to edit my config so often. Thanks for this great tool @tilfin and for the PR @micaelgfilipe !

**Comment by tilfin** [comment]
I close it for the reason is https://github.com/tilfin/aws-extend-switch-roles/issues/110#issuecomment-524108459

**Pull request closed without merging**